src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+const people = [...document.querySelectorAll('li')];
+
+function sortBySalary(arr) {
+  const salaries = arr.map((person) => {
+    const salary = person.getAttribute('data-salary');
+
+    return +salary.replace(/[$,]/g, '');
+  });
+
+  return salaries.sort((person1, person2) => person2 - person1);
+}
```

Comment:
The `sortBySalary` function currently returns an array of sorted salaries, not the original array of people sorted by salary. To fix this, you should sort the `arr` directly based on the salary attribute and return the sorted array of people.